index.html

```diff
@@ -4,10 +4,38 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio website </title>
 </head>
 <body>
-    <h1>My portfolio website</h1>
+    <main class="Portfolio-container">
+        <div class="header">
+            <div class="logo">RIDA</div>
+              <div class="menu-container">
+                <img src="Images/icon-menu.png" alt="menu-icon">
+              </div>
+            </div>
+            <div class="headline-section">
+                <div class="primary-text"> 
+                    <h1>I’m RIDA <br> Glad to see <br> you!</h1>
+                    <p>
+                        I’m a software developer! I can help you build a product
+                         , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <div class="supportive-text">
+                        LET’S CONNECT        
+                    </div>
+                    <div class="social-media-icons">
+                        <img src="Images/social media icons.svg" alt="Social media icons">
+                    </div>
+                </div>
+               
+            </div>
```

Comment:
The same happens here, the style is good, but there are some things to take in mind:

- [ ] HTML provides a semantic tag which is `<section>` please consider using it over the `<div>` as it provides more information about the content to the browser. 🚧 
- [ ] The "Let's Connect" text could be better wrapped inside an `<hx>` tag since it's acting as a title for the social media links.
- [ ] Please consider adding your social media icons one by one, this is because the purpose of them is to provide an easy way to link people that see your portfolio to your social media and make it easier for them to connect with you! 🚧 

You could use this structure as a hint: 💡 
```html
<ul>
  <li>
    <a href="yourgithub.com"> <img src="github_icon" /> </a>
  </li>
  <li>
    <a href="yourlinkedin.com"> <img src="linkedin_icon" /> </a>
  </li>
...
</ul>

```

Please consider applying these changes! 🤓 🛠️